Makefile

```diff
@@ -0,0 +1,4 @@
+fuzz:
```

Comment:
Do we really need to commit this Makefile?

Reply:
Not actually

Reply:
Should we run them as part of the CI?

Reply:
>Should we run them as part of the CI? 

My proposal is a little bit longer but it's better for the DHCP project.
See, the DHCP project is used by a lot of [projects](https://github.com/insomniacslk/dhcp#public-projects-that-use-it), so I can integrate this into [oss-fuzz](https://google.github.io/oss-fuzz/) and later [CIFuzz](https://google.github.io/oss-fuzz/getting-started/continuous-integration/) can be added.

Steps are:
* Merge this PR.
* Integrate oss-fuzz.
* Implement CIFuzz for DHCP PR.